fcrepo-clients/src/main/java/edu/unc/lib/dl/persist/api/ingest/DepositData.java

```diff
@@ -275,4 +276,16 @@ public boolean getOverrideTimestamps() {
     public void setOverrideTimestamps(boolean overrideTimestamps) {
         this.overrideTimestamps = overrideTimestamps;
     }
+
+    /**
+     * @return if true, then if this deposit would create a parent container to wrap
```

Comment:
Is this note write? It seems like if it's true it should create a folder.

Reply:
You're right, I flipped the direction of the flag halfway through working on it. I'll update